app/views/ExploreDeep/MapView/index.tsx

```diff
@@ -259,6 +228,40 @@ function ExploreDeepMapView(props: Props) {
         setClusterClicked(false);
     }, []);
 
+    const createClusterMarker = useCallback((markerProps: object) => {
+        const {
+            project_ids: projectIds,
+        } = markerProps as { project_ids: string | undefined };
```

Comment:
```suggestion
        } = markerProps as ClusterProperties;
```

Shouldn't this just be ClusterProperties?